### ISSUE TITLE
Specification

### ISSUE THREAD

**Issue by joshuaavalon** [gboudreau/XBMCnfoTVImporter.bundle#90]
What is the specification of the xml? I am hand crafting the xml. What are the necessary field, mapping field and sample data? It would be nice if you can provide the specification in file or wiki

**Comment by Em31Et** [comment]
You can use these tools to generate compatible nfo.
[http://kodi.wiki/view/Library_managers](url)
Since Plex have less fields, not all fields in the nfo are used.

**Comment by SlrG** [comment]
IMHO the source code is quite readable. Look for lines which contain nfoXML e.g.
try: metadata.title = nfoXML.xpath("title")[0].text
The title tag is used.

**Comment by joshuaavalon** [comment]
@SlrG Yes, I can read source code. But it is like documentation. Not necessary but good to have.

**Comment by SlrG** [comment]
Foundation to a wiki is created. Maybe I can add some info about this there later on.

**Comment by otravers** [comment]
> You can use these tools to generate compatible nfo.
> [http://kodi.wiki/view/Library_managers](url)
> Since Plex have less fields, not all fields in the nfo are used.

That link no longer works. The best alternative is probably:
https://kodi.wiki/view/NFO_files
